packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayerContainer.js

```diff
@@ -16,24 +28,92 @@ function storeMapper (stores) {
 class InteractionLayerContainer extends Component {
   constructor () {
     super()
+
+    this.state = {
+      currentMark: null,
+      drawing: false,
+      markings: []
```

Comment:
Maybe instead of an array we can use a map? That way you can leverage the ids of each mark to get/set as long as they're unique ids?

Reply:
Why `markings` and not `marks`?